lib/ex_unit/lib/ex_unit/case.ex

```diff
@@ -431,11 +434,12 @@ defmodule ExUnit.Case do
     tag = Module.delete_attribute(mod, :tag)
     async = Module.get_attribute(mod, :ex_unit_async)
 
-    {name, describe, describetag} =
+    {name, describe, describe_line, describetag} =
       if describe = Module.get_attribute(mod, :ex_unit_describe) do
```

Comment:
If you change ex_unit_describe to store `{line, describe}`, this `if` needs to be come a `case`. :)